### PR TITLE
Automate EDA - Education Cloud Settings Test Suite

### PR DESCRIPTION
**Pull request by xinnanlisf**
In addition to new test cases, I have also: 

- Added a new qa locator to eda settings page title
- Cleaned up irrelevant cases in edc settings.
- Renamed test suite to edc settings page. 

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by salvadorsfdc — CHANGES_REQUESTED** [review]
changes look great, just some minor comments.

File: robot/EDA/tests/browser/settings/home/edc_experience.robot [hunk 1]
```diff
@@ -1,36 +0,0 @@
...
-Resource        robot/EDA/resources/EDA.robot
-Library         cumulusci.robotframework.PageObjects
-...             robot/EDA/resources/EducationCloudSettingsPageObject.py
-...             robot/EDA/resources/ReleaseManagementPageObject.py
-
-Suite Setup     Open Test Browser
-Suite Teardown  Capture screenshot and delete records and close browser
-
-*** Test Cases ***
-Verify education cloud settings app is displayed
```
[the remark this answers is not in the document]
Comment (xinnanlisf) [reply robot/EDA/tests/browser/settings/home/edc_experience.robot:12]: yes i think this test case is quite irrelevant. Why would we test it in edc settings page?

File: robot/EDA/tests/browser/settings/home/edc_experience.robot [hunk 2]
```diff
@@ -1,36 +0,0 @@
...
-    [Documentation]         Validates 'Education Cloud Settings" app is displayed under app tile.
-    [tags]                  rbt:high        W-9549044
-    Open app launcher
-    Verify item exists       Education Cloud Settings
-
-Verify products tools and resources tiles are displayed
```
[the remark this answers is not in the document]
Comment (xinnanlisf) [reply robot/EDA/tests/browser/settings/home/edc_experience.robot:18]: that's a good catch. I will update them.

File: robot/EDA/tests/browser/settings/home/edc_experience.robot [hunk 3]
```diff
@@ -1,36 +0,0 @@
...
-    Go to education cloud settings home
-    Current page should be                  Home        Education Cloud Settings
-    Verify app tiles displayed
-    ...                                     Products=Education Data Architecture
-    ...                                     Tools=Settings Health Check
-    ...                                     Resources=Trailhead
-    ...                                     Resources=Trailblazer Community
-    ...                                     Resources=YouTube
-
-Verify release management page is displayed when user clicks on go to release management button
```
[the remark this answers is not in the document]
Comment (xinnanlisf) [reply robot/EDA/tests/browser/settings/home/edc_experience.robot:30]: that's a good catch. I will update them.

File: robot/EDA/tests/browser/settings/home/edc_experience.robot (hunk 2 above)
[the remark this answers is not in the document]
Comment (xinnanlisf) [reply robot/EDA/tests/browser/settings/home/edc_experience.robot:18]: I can't quite find the ts. Maybe you are referring to the one Kris is working on?

File: robot/EDA/tests/browser/settings/home/edc_experience.robot (hunk 1 above)
[the remark this answers is not in the document]
Comment (salvadorsfdc) [reply robot/EDA/tests/browser/settings/home/edc_experience.robot:12]: yeah, it is weird, I didn't find a TS related to it anyways.

**Review by salvadorsfdc — APPROVED** [review]
approving changes

**Pull request merged**